phpunit.xml.dist

```diff
@@ -37,5 +37,8 @@ It's auto-generated by sonata-project/dev-kit package.
     <php>
         <ini name="precision" value="8" />
         <env name="SYMFONY_DEPRECATIONS_HELPER" value="max[self]=0" />
+        <env name="KERNEL_CLASS" value="\Sonata\PageBundle\Tests\App\AppKernel" />
```

Comment:
hey @jordisala1991  I tried to do this before But @VincentLanglet mentioned  that this file is generated automatically

Reply:
He plans to update the dev-kit file generator